portfolio/src/main/webapp/script.js

```diff
@@ -244,3 +244,37 @@ function closeWindows(map) {
     infoWindows[i].close(map, markers[i]);
   }
 }
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Task');
+  data.addColumn('number', 'Minutes');
+  data.addRows([
+    ['Deciding when to practice', 10],
+    ['Waiting to get picked up', 25],
+    ['Driving to/from where we\'re practicing', 20],
+    ['Waiting for everyone to show up', 30],
+    ['Socializing', 30],
+    ['Setting up gear', 15],
+    ['Playing songs that we aren\'t playing at our next gig', 20],
+    ['Deciding our set list', 30],
+    ['Rehearsing', 120],
+    ['Creating harmonies', 15],
+    ['Messing up an entrance or skipping a section', 25]
```

Comment:
ls this one band practice or multiple? The minutes add up to a large amount of time... 
lf it's multiple, might want to change the title of the pie chart?

Reply:
It's one band practice. Long nights.